src/openforms/appointments/base.py

```diff
@@ -220,3 +226,20 @@ def get_appointment_details_html(self, identifier: str) -> str:
         return render_to_string(
             "appointments/appointment_details.html", {"appointment": details}
         )
+
+    def get_appointment_links(self, submission: Submission) -> Dict[str, str]:
+
+        token = submission_appointment_token_generator.make_token(submission)
+
+        cancel_uri = reverse(
+            "appointments:appointments-verify-cancel-appointment-link",
+            kwargs={
+                "token": token,
+                "base64_submission_uuid": urlsafe_base64_encode(
+                    str(submission.uuid).encode()
+                ),
+            },
+        )
+        cancel_url = f"{settings.BASE_URL}{cancel_uri}"
```

Comment:
Is this the best way to build the full url?  Since we don't have access to the request here I was hoping we can just use this settings variable.